mtest/libmscore/measure/tst_measure.cpp

```diff
@@ -459,6 +460,42 @@ void TestMeasure::checkMeasure()
       delete score;
       }
 
+//---------------------------------------------------------
+///   undoDelInitialVBox_269919
+///    1. Delete first VBox
+///    2. Change duration of first chordrest
+///    3. Undo to restore first chordrest
+///    4. Undo to restore initial VBox results in assert failure crash
+//---------------------------------------------------------
+
+void TestMeasure::undoDelInitialVBox_269919()
+      {
+      MasterScore* score = readScore(DIR + "undoDelInitialVBox_269919.mscx");
```

Comment:
It would be the only place in the code where it's used (except importmidi which is from a different author than most of the code). Personally, I had to look it up... It would save a "delete" but make the code less clear (at least to me...)

Reply:
So, let's find time to shift our point of view from C to C++ again :)

Reply:
Let's discuss this somewhere else indeed. 
`delete` is definitely C++, not C. By using "fancier" C++ just for the sake of it, we might make the code (even) less readable that what it is right now and this is not good thing.